src/Stratis.Bitcoin.Features.RPC/RPCFeature.cs

```diff
@@ -49,6 +50,21 @@ public static void PrintHelp(Network network)
             RpcSettings.PrintHelp(network);
         }
 
+        /// <summary>
+        /// Get the default configuration.
+        /// </summary>
+        /// <param name="builder">The string builder to add the settings to.</param>
+        public static void GetDefaultConfiguration(StringBuilder builder)
```

Comment:
Maybe call this `BuildDefaultConfigurationFile`.

Reply:
Changed.